tools/circt-test/circt-test.cpp

```diff
@@ -66,6 +69,12 @@ Options opts;
 // Tool Implementation
 //===----------------------------------------------------------------------===//
 
+bool shouldIgnore(verif::FormalOp formalOp, Options &opts) {
+  return formalOp->hasAttrOfType<BoolAttr>("ignore") &&
+         cast<BoolAttr>(formalOp->getAttr("ignore")).getValue() &&
```

Comment:
Or, to avoid double lookup:
```
if (auto boolAttr = formalOp->getAttrOfType<BoolAttr>("ignore"))
  return boolAttr.getValue() && !opts.includeIgnored;

return false;
```